.github/workflows/core-daily.yaml

```diff
@@ -0,0 +1,311 @@
+# Runs every day. Tests TRC-2 in ALL kernels.
```

Comment:
I'd rename file to test-core-daily.yaml so we know this workflow is all about testing